src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -103,3 +104,20 @@
 {% if swiss_qrbill %}
 <link rel="stylesheet" href="{% static "pretixplugins/banktransfer/swisscross.css" %}">
 {% endif %}
+
+{% if has_invoices %}
+    <hr>
+    <form class="form-inline helper-display-inline" method="post"
+            action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
+        {% csrf_token %}
+        <p>
+            {% blocktrans trimmed %}
+                To send the invoice directly to your accounting department, please enter their email address:
+            {% endblocktrans %}
+        </p>
+        <label for="mail_invoice_email" class="sr-only">{% trans "Invoice recipient email" %}:</label>
+        <input type="email" name="email" id="mail_invoice_email" class="form-control" value="" required />
+        <button class="btn btn-default">{% trans "Send invoice via email" %}</button>
+    </form>
```

Comment:
Here's a suggestion that I think looks a little cleaner, but it's probably a matter of taste – take it or leave it :)

```suggestion
    <form method="post" action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
        {% csrf_token %}
        <p>
            {% blocktrans trimmed %}
                To send the invoice directly to your accounting department, please enter their email address:
            {% endblocktrans %}
        </p>
        <div class="row">
            <div class="col-md-9 col-xs-12">
                <label for="mail_invoice_email" class="sr-only">{% trans "Invoice recipient email" %}:</label>
                <input type="email" name="email" id="mail_invoice_email" class="form-control" value="" required
                    placeholder="{% trans "Email address" %}" />
            </div>
            <div class="col-md-3 col-xs-12">
                <button class="btn btn-default btn-block">
                    <span class="fa fa-envelope-o" aria-hidden="true"></span>
                    {% trans "Send invoice via email" %}
                </button>
            </div>
        </div>
    </form>
```

Reply:
I think this way we can even get rid of one of the `<hr>`s

![grafik](https://github.com/pretix/pretix/assets/388142/8f4997f8-3605-4cbe-95d9-ce18ff57975e)

Reply:
Yep, agreed, the hr is not necessary then